src/model/database/db_build.sql

```diff
@@ -0,0 +1,33 @@
+BEGIN; 
+
+DROP TABLE IF EXISTS users , reservations , rooms CASCADE ;
+
+CREATE TABLE users(
+    id serial primary key ,
+    first_name VARCHAR(20) NOT NULL,
+    last_name VARCHAR(20) NOT NULL,
+    email_address VARCHAR(50) UNIQUE NOT NULL  ,
+    phone_num INTEGER UNIQUE NOT NULL,
+    status VARCHAR(10) NOT NULL
```

Comment:
I think it would be more convenient to use `ENUM` types to constraint the status to its three states,
check https://www.postgresql.org/docs/9.1/static/datatype-enum.html

this is an example of how we did it before:
```
DROP TYPE IF EXISTS seen, states;
CREATE TYPE states AS ENUM ('approved', 'pending', 'decline');
CREATE TYPE seen AS ENUM ('yes', 'no');
```
then you can use it as any SQL type